ctags/parsers/pascal.c

```diff
@@ -44,15 +44,11 @@ static void createPascalTag (tagEntryInfo* const tag,
 {
 	if (PascalKinds [kind].enabled  &&  name != NULL  &&  vStringLength (name) > 0)
 	{
-		initTagEntry (tag, vStringValue (name));
+		initTagEntry (tag, vStringValue (name), &(PascalKinds [kind]));
 
-		tag->kindName = PascalKinds [kind].name;
-		tag->kind     = PascalKinds [kind].letter;
 		tag->extensionFields.signature = arglist;
 		tag->extensionFields.varType = vartype;
 	}
-	else
-		initTagEntry (tag, NULL);
```

Comment:
Ah right, I somehow thought this was makeTagEntry() and was wondering why there's a tag created with a NULL name (and name is then later checked in makePascalTag() and based on that the tag is created or not). I'll use the code from uctags.